python/b2gedmntuples_cff.py

```diff
@@ -1149,11 +1149,22 @@
 jetKeysAK4.jetLabel = cms.InputTag("jetUserData")
```

Comment:
I think it is better to rename these as jetsAK4CHS to keep parity with PUPPI jets naming convention